src/components/Main/Main.js

```diff
@@ -1,10 +1,30 @@
 import React from "react";
+import { Route, Switch } from 'react-router';
+
+import './Main.scss';
+import AdminOrders from '../../pages/AdminOrders';
+import AdminProducts from '../../pages/AdminProducts';
+import CheckOut from '../../pages/CheckOut';
+import Home from '../../pages/Home';
+import Orders from '../../pages/Orders';
+import OrderSuccess from '../../pages/OrderSuccess';
+import Products from '../../pages/Products';
+import ShoppingCart from '../../pages/ShoppingCart';
 
 function Main() {
     return (
-        <main>
-            <h2>This is the Main Component</h2>
-        </main>
+        <div className="main">
+            <Switch>
+                <Route exact path="/checkout" component={CheckOut}></Route>
+                <Route exact path="/admin-products" component={AdminProducts}></Route>
```

Comment:
The Route tag can be self closing since nothing is going between the open and close tags